### PR TITLE
remove type if current action is not creation

### PR DESCRIPTION
**Pull request by Vesax**
I am targetting this branch, becaus this is non-BC fix

## Changelog

``` markdown
### Fixed
- New block type is taken from previous one
```
## Subject

When we are adding a block and clicking Actions > Add new - type of current block will be saved in url and user won't be asked for it.
If we save changes using "Update and close" button, we will be redirected to list page with type parameter, and it also will be saved in url if we will try to create new block (and we won't be asked for type).

### CODE REVIEW

**Comment by Vesax** [comment]
@OskarStark, is it okay now?

**Comment by OskarStark** [comment]
Travis failed

File: Tests/Model/Block.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\PageBundle\Tests\Model;
```
Comment (core23) [inline Tests/Model/Block.php]: Please use the `Fixtures` namespace.

File: Tests/Model/Block.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+/**
+ * @author Artur Vesker <arturvesker@gmail.com>
+ */
+class Block extends \Sonata\PageBundle\Model\Block
```
Comment (core23) [inline Tests/Model/Block.php]: `DummyBlock` insteaf of just `Block`?

Comment (Vesax) [inline Tests/Model/Block.php]: I created this based on Page model for tests

**Comment by core23** [comment]
Ping @Vesax

**Comment by core23** [comment]
Please rebase @Vesax

**Comment by core23** [comment]
This PR looks dead. Feel free to reopen if you will continue working on this.

**Pull request closed without merging**